juanda-sanchon/challenge/heroes/jss/script.js

```diff
@@ -0,0 +1,42 @@
+let body = document.querySelector('body');
+
+function createheader(){
+    let header = document.createElement('header');
+    let title = document.createElement('h1');
+    title.textContent ="Tour of Heroes";
+    header.appendChild(title);
+    let list = document.createElement('ul');
+    //let objList;
+    //let link;
+    for(let i=0 ; i<2 ; i++){
+        let objList = document.createElement('li');
```

Comment:
no seria mejor extraer la creación del enlace a una funcion que sea createLink en vez de poner ese if en el for? 

Imagina que tienes 4 enlaces.. tendrias que poner 4 if...